src/workflow/RunEnergyPlus.cpp

```diff
@@ -157,7 +157,7 @@ void OSWorkflow::runEnergyPlus() {
     // TODO: workflow-gem was manually running expandObjects prior to the potential serialization to json
     // Should we rather pass -x to the E+ cmd line?
     if (!workflowJSON.runOptions()->skipExpandObjects()) {
-      const std::string cmd = openstudio::toString(runDirResults.expandObjectsExe.native());
+      const std::string cmd = fmt::format("\"{}\"", openstudio::toString(runDirResults.expandObjectsExe.native()));
```

Comment:
expandObjectsExe is infered from getEnergyPlusDirectory which uses getEnergyPlusExecutable which will use ENERGYPLUS_EXE_PATH (as long as it's not running from the build dir)